modules/web/src/app/cluster/details/cluster/rbac/dialog/add-binding/component.ts

```diff
@@ -179,6 +180,9 @@ export class AddBindingComponent implements OnInit, OnDestroy {
           `Added the ${binding.subjects[binding.subjects.length - 1].name} cluster binding`
         )
       : this._notificationService.success(`Added the ${binding.subjects[binding.subjects.length - 1].name} binding`);
+    this._clusterServiceAccountService.update();
+    this._rbacService.refreshClusterBindings();
+    this._rbacService.refreshNamespaceBindings();
```

Comment:
I think we're already handling these updates in parent component (which this dialog is opened).